cockatrice/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ SET(cockatrice_SOURCES
     src/dlg_edit_password.cpp
     src/dlg_edit_tokens.cpp
     src/dlg_edit_user.cpp
+	src/dlg_forgotpassword.cpp
```

Comment:
Ah yes, I forgot to change that setting in VS when I reloaded it.  Ill roll through and change tab->spaces after all the reviewing is done before committing it so to not clutter up things during code review.